### ISSUE TITLE
Unstable loss

### ISSUE THREAD

**Issue by Diligentfight** [huochaitiantang/pytorch-deep-image-matting#4]
Have you meet unstable loss problem?

**Comment by huochaitiantang** [comment]
The training loss is always stable. Please set the learning rate to 1e-5 in case of gradient explosion.
